sdk/keyvault/test-resources.json

```diff
@@ -127,18 +127,6 @@
         }
     ],
     "outputs": {
-        "AZURE_TENANT_ID": {
```

Comment:
With everything else I've seen, this *should* work but please try for both CI and dev workflows. I.e. if you follow the instructions I wrote in eng/common/TestResources/README.md, do live tests (maybe just run one) still work?

Reply:
Yes, checking that is on my list.